### DISCUSSION TITLE
How can I now exclude some tests from command line on Cypress 10

### DISCUSSION THREAD

**Discussion by ztulic** [cypress-io/cypress#22249] [Questions and Help]
Before Cypress 10 I used to exclude some tests using this command:
`npm run cy:run -- --browser chrome --headless --spec "**/!(test1.js|test2.js)"`

All tests except for test1.js and test2.js will run, but now this is not working.
Is there any way to do that in new Cypress version?

Thanks in advance.

**Comment by j1000** [comment]
I believe this feature is unchanged. Perhaps you have stumbled upon a bug?

**Reply by ztulic** [reply]
Thank for the reply, I have opened a bug for this:
https://github.com/cypress-io/cypress/issues/22407

**Comment by melibe23** [comment]
I have the same issue, were you able to make it work?

**Reply by ztulic** [reply]
Yes I am, with this:
https://github.com/cypress-io/cypress/issues/22407#issuecomment-1212558547
